packages/gasket-typescript-tests/test/resolve.spec.ts

```diff
@@ -0,0 +1,159 @@
+import {
+  pluginIdentifier,
+  presetIdentifier,
+  Loader,
+  Resolver,
+  PluginInfo,
+  PresetInfo,
+  ModuleInfo
+} from '@gasket/resolve';
+
+describe.only('@gasket/resolve', () => {
+  describe('pluginIdentifier', function () {
+    it('exposes expected member types', function () {
+      const id = pluginIdentifier('@gasket/plugin-example');
+
+      const rawName: string = id.rawName;
+      const fullName: string = id.fullName;
+      const longName: string = id.longName;
+      const shortName: string = id.shortName;
+      const name: string = id.name;
+      const version: string | null = id.version;
+      const full: string = id.full;
+      const isShort: string = id.isShort;
+      const isLong: string = id.isLong;
+      const isPrefixed: boolean = id.isPrefixed;
+      const hasScope: boolean = id.hasScope;
+      const hasProjectScope: boolean = id.hasProjectScope;
+      const hasVersion: boolean = id.hasVersion;
+
+      // @ts-ignore-error
```

Comment:
If you want to test that TS catches errors, you should use `@ts-expect-error` instead. That will allow the TS error _and_ cause a failure if TS doesn't raise a compiler error.

Reply:
Aha, yes thank you